### ISSUE TITLE
Self hosted  Gitlab configuration issue

### ISSUE THREAD

**Issue by avicoder** [mergestat/mergestat#1155]
Hi 

Could you tell me where to look since I couldn't find how to configure the token for a self hosted gitlab instance?
<img width="474" alt="image" src="https://github.com/mergestat/mergestat/assets/2093260/fb7b8c61-b3ba-411d-a331-8b22da98f7c0">

**Comment by matnav** [comment]
Certainly! Here's a more concise version that leverages GitHub's automatic linking for issues:

Hi @avicoder,

I've been working on a workaround for configuring Mergestat with self-hosted GitLab, which might be useful for you. It involves setting a custom base URL and ensuring the access token is correctly used. I've detailed this approach in issue #1161, where I also link to a fork with the applied changes.

This temporary solution could help while we await a more integrated feature from the devs. Check out issue #1161 for more details, and don't hesitate to ask if you have questions!

Thanks,
matnav

**Comment by matnav** [comment]
@avicoder, 
 I've been working on a workaround for configuring Mergestat with self-hosted GitLab, which you might find useful. It involves setting a custom base URL for the GitLab API and ensuring the access token is properly utilized. All the details of this approach have been outlined in issue #1161, along with a link to a fork where I've applied these changes.

This workaround could serve as a temporary fix while we await a more integrated solution from the Mergestat developers. Feel free to check out issue #1161 for more information. If you have any questions or need further clarification, I'm more than happy to help!
